src/vaults/utils.ts

```diff
@@ -55,8 +55,8 @@ async function* readDirRecursively(
   fs,
   dir = '.',
 ): AsyncGenerator<string, void, void> {
-  const dirents = await fs.promises.readdir(dir);
-  for (const dirent of dirents) {
+  const dirEntities = await fs.promises.readdir(dir);
+  for (const dirent of dirEntities) {
```

Comment:
I believe this is `dirEntries` and `dirEntry`.

Reply:
Fixed.